src/SDKs/IotHub/Management.IotHub/Properties/AssemblyInfo.cs

```diff
@@ -14,5 +14,5 @@
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en")]
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```

Comment:
@anusapan Please do not update the AssemblyVersion here, only AssemblyFileVersion must be updated for minor version updates

Reply:
Done